server.js

```diff
@@ -193,6 +193,9 @@ const questionForm = pug.compileFile('views/question-creation.pug');
 const questionEdit = pug.compileFile('views/question-edit.pug');
 const statistics = pug.compileFile('views/statistics.pug');
 
+const regexForm = pug.compileFile('views/regex-answer.pug');
+const multiplechoiceFrom = pug.compileFile('views/mc-answer.pug');
```

Comment:
Make it camel case and fix the spelling (from instead of form) to reduce confusion

Reply:
fixed